core/src/main/java/jenkins/model/lazy/SortedIntList.java

```diff
@@ -131,7 +131,7 @@ public int ceil(int v) {
     }
 
     public boolean isInRange(int idx) {
-        return 0<=idx && idx<size;
+    	return 0<=idx && idx<size;
```

Comment:
Strange....
This line is indented correctly.
Look at: https://github.com/brunocarneiro/jenkins/blob/proposeNewNumberAlreadyInUse/core/src/main/java/jenkins/model/lazy/SortedIntList.java